kustomization.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+images:
+# Controller
+- name: objectstorage-controller
+  newName: quay.io/containerobjectstorage/objectstorage-controller
+  newTag: latest
+# Sidecar
+- name: sample-driver
+  newName: quay.io/containerobjectstorage/sample-driver
+  newTag: latest
+- name: object-storage-sidecar
+  newName: quay.io/containerobjectstorage/object-storage-sidecar
+  newTag: latest
+
+resources:
+# CRDs
+- https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-api/master/crds/objectstorage.k8s.io_bucketaccessclasses.yaml
```

Comment:
I'm personally not super fond of `kustomization`s which reference manifests from other repositories. Can't these be deployed as a prerequisite?
Over time, it could make sense to ship these with a `cosi-controller-manager` release directly: in the end, the `cosi-controller-manager` is a singleton in a cluster, and as such it defines which CRD/API (versions) are supported in the cluster. Hence, its installation could enforce/deploy these.

Reply:
We will revisit this at a later point in time. Captured this in issue https://github.com/kubernetes-sigs/container-object-storage-interface-controller/issues/21